CHANGELOG.md

```diff
@@ -9,6 +9,21 @@
 - Test Ref resources to IAM Roles have good assume role documents.  Example: Lambda Function Ref refers to an IAM Role that can be assume by Lambda.
 - More Warnings around hard coded values (Regions, AccountIds) to help with the practice of reusability
 
+### 0.5.0
+###### Features
+- Load all instances of CloudFormationLintRule in a file.  Class doesn't need to match filename.
+- Allow load yaml to accept a string allowing people to use cfn-lint as a module
+- Add rule W6001 to test outputs that are just using an import value
+- Update specs to ones released on August 10, 2018
+###### Fixes
+- Update E2507 to support conditions and using get_values to test all condition paths
```

Comment:
Is some work, but you can make links tot the rule in the documentation ;)

Like this: [E2507](/docs/rules.md#E2507)